.claude/scripts/portfolio-surveyor.test.sh

```
@@ -237,6 +237,20 @@ grep -Fq 'authored by an exact dependency-automation' "${product_engineering_ski
+# The digest must carry an explicit measured|unknown grammar, forbid emitting a count from one
+# page, and prefer unknown under budget pressure — otherwise every survey re-improvises the metric
+# and a truncated census looks complete.
+# Literal Markdown code spans; command substitution is intentionally disabled.
+# shellcheck disable=SC2016
+grep -Fq 'board_coverage=<measured:' "${surveyor}" ||
+  fail "surveyor digest has no board_coverage measured|unknown grammar"
+grep -Fq 'never emit a count from a single page' "${surveyor}" ||
+  fail "surveyor may still emit a board-coverage count from a single-page read"
+grep -Fq 'board_coverage=unknown' "${surveyor}" ||
+  fail "surveyor has no unknown token for a truncated or budget-limited board census"
+grep -Fq 'prefer `unknown` over a partial number' "${surveyor}" ||
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)